.gitignore

```diff
@@ -1,12 +1,23 @@
+# --- REMOVE THIS BLOCK! --- #
+composer.lock
+yarn.lock
```

Comment:
I think we can just remove this here. If we commit the `package-lock.json` we can pin to specific versions OOTB to avoid issues like that Highlight.js one we had at the beginning ot the week. A project can always update the dependencies themselves, but at that point it's on them and not us.